hack/update-import-aliases.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${SCRIPT_ROOT}"
+ROOT_PATH=$(pwd)
+
+temp_path=$(mktemp -d)
+pushd "${temp_path}" >/dev/null
+cp "${ROOT_PATH}"/go.mod .
+GO111MODULE=on go get "k8s.io/kubernetes@v1.21.3"
```

Comment:
```suggestion
GO111MODULE=on go get "k8s.io/kubernetes/cmd/preferredimports@v1.21.3"
```
`go get "k8s.io/kubernetes@v1.21.3"` will install nothing.

Same as below.

Reply:
OK.